2.3.x+lxd support

g-bougard opened the pull request:
Experimental lxd support

g-bougard commented:
Also related to [forge issue 3026](http://forge.fusioninventory.org/issues/3026)

guillomovitch commented:
Even if that's just comment, I'd rather not have individual authors or license per file. The whole license is GPLv2, no need to reassert it, and we have either README or THANKS file for contributers.

g-bougard commented:
Agree, just made updates in e8f308be
Also updated other stuffs there to be more up-to-date: moved @goneri in THANKS, added @devtom30 and updated Teclib Editions URL
